packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -333,9 +328,13 @@ export class Summarizer extends EventEmitter implements ISummarizer {
             const coordinatorCreateP = this.runCoordinatorCreateFn(this.runtime);
 
             coordinatorCreateP.then((runCoordinator) => {
+                const newConfig = {
+                    ...this.configurationGetter(),
+                };
+                newConfig.state = "disableHeuristics";
```

Comment:
There has to be a better way to do it. Thoughts ?

Reply:
const newConfig = {
                    ...this.configurationGetter(),
                    state: "disableHeuristics",
                };

You can even inline it if you want.

Reply:
That was the first thing I tried but got an error: 
src/summarizer.ts:338:83 - error TS2345: Argument of type '{ state: string; } | { state: string; initialSummarizerDelayMs: number; summarizerClientElection: boolean; maxAckWaitTime: number; maxOpsSinceLastSummary: number; } | { state: string; idleTime: number; ... 6 more ...; maxOpsSinceLastSummary: number; }' is not assignable to parameter of type '{ state: "disabled"; } | ({ state: "disableHeuristics"; } & ISummaryConfigurationBaseSettings) | ({ state: "enabled"; } & ISummaryConfigurationHeuristicSettings) | undefined'.
[build:compile] [build:commonjs]   Type '{ state: string; }' is not assignable to type '{ state: "disabled"; } | ({ state: "disableHeuristics"; } & ISummaryConfigurationBaseSettings) | ({ state: "enabled"; } & ISummaryConfigurationHeuristicSettings) | undefined'.
[build:compile] [build:commonjs]     Type '{ state: string; }' is not assignable to type '{ state: "enabled"; } & ISummaryConfigurationHeuristicSettings'.
[build:compile] [build:commonjs]       Type '{ state: string; }' is not assignable to type '{ state: "enabled"; }'.
[build:compile] [build:commonjs]         Types of property 'state' are incompatible.
[build:compile] [build:commonjs]           Type 'string' is not assignable to type '"enabled"'.

Reply:
Can you try this:

```ts
const newConfig: ISummaryConfiguration = {
   ...this.configurationGetter(),
    state: "disableHeuristics",
};
```

or 

```ts
const newConfig: ISummaryConfiguration = {
   ...this.configurationGetter(),
   state: "disableHeuristics",
} as ISummaryConfiguration;
```


I.d. add type info? There should be a way to do it right :) The difference I see is only in type info.

Reply:
I had to add: 

// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
                const newConfig: ISummaryConfiguration = {
                    ...this.configurationGetter(),
                    state: "disableHeuristics",
                } as ISummaryConfiguration;